pkg/executor/aggregate/agg_spill.go

```diff
@@ -25,34 +32,250 @@ import (
 // maxSpillTimes indicates how many times the data can spill at most.
 const maxSpillTimes = 10
 
+const spillFlag = 1
+const isSpillingFlag = 2
+const hasErrorFlag = 3
+const partialStageFlag = 4
+const spilledPartitionNum = 256
+const spillTasksDoneFlag = -1
+
+type parallelHashAggSpillHelper struct {
+	lock                        sync.Mutex
+	diskTracker                 *disk.Tracker
```

Comment:
Can we use `if diskTracker != 0` and remove `isTrackerEnabled`

Reply:
> Can we use `if diskTracker != 0` and remove `isTrackerEnabled`

okk, I will change it.